README.md

```diff
@@ -95,6 +95,18 @@ Read all visible cookies:
 Cookies.get(); // => { name: 'value' }
 ```
 
+*Note: It is not possible to read a particular cookie by passing one of the cookie attributes (which may or may not
```

Comment:
"It is not possible to `.get()` a particular cookie..."?

Reply:
In plain text I don't see what's wrong with using "read"... that's how the operation is usually referred to imo.

Reply:
That's fine, just wondering if it would make more sense. LGTM